plugins/todo-backend/src/schema/openapi.yaml

```diff
@@ -0,0 +1,127 @@
+openapi: 3.1.0
+
+info:
+  title: '@backstage/plugin-todo-backend'
+  version: '1'
+  description: The Backstage backend plugin that provides source code todo comment browsing.
+  license:
+    name: Apache-2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact: {}
+
+servers:
+  - url: /
+
+components:
+  examples: {}
+  headers: {}
+  parameters: {}
+  requestBodies: {}
+  responses: {}
+  schemas: {}
+  securitySchemes:
+    JWT:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+paths:
+  /v1/todos:
+    get:
+      operationId: ListTodos
+      responses:
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
```

Comment:
nit: I generally pull out the response body into it's own reference. That's probably extreme, but I think pulling out the array objects at least would make sense and then reference them here.

Also, it might make sense for us to set up a basic style guide/linter for OpenAPI specs. I found https://docs.stoplight.io/docs/spectral/9ffa04e052cc1-spectral-cli and https://github.com/apisyouwonthate/style-guide which seems very powerful.

Reply:
yep didn't feel there was a need for a lot of abstraction in this one, but pulled out the todo items